Layout_index_v1.html

```diff
@@ -18,63 +18,86 @@
 		<div class="grid-container">
 			<div>
 				
-		<div class="header">Home Page</div>
+	    <div class="header"><img src="images/LOGO2.jpg" alt="iMages"></div>
 			
 		<div>
 			
 	<!--This class is for my navigation/menu-->
     <div class="nav">
 		<ul>
-			<li><a href="index_v1.html">Home | </a></li>
-			<li><a href="CommunityService.v1.html">Community service | </a></li>
+			<li><a href="index.v2.html">Home | </a></li>
+			<li><a href="CommunityService.v2.html">Community service | </a></li>
 			<li><a href>Outdoor activities |</a>
 			    <ul>
-					<li><a href="indexpage2.v1.html">Adrenaline Forest and Te Rauparaha Arena</a>
-					<li><a href="indexpage3.v1.html">Rangituhi Walk</a>
-					<li><a href="indexpage4.v1.html">Wainuiomata Camp pt1</a>
-					<li><a href="indexpage5.v1.html">Wainuiomata Camp pt2</a>
+					<li><a href="indexpage2.v2.html">Adrenaline Forest and Te Rauparaha Arena</a>
+					<li><a href="indexpage3.v2.html">Rangituhi Walk</a>
+					<li><a href="indexpage4.v2.html">Wainuiomata Camp pt1</a>
+					<li><a href="indexpage5.v2.html">Wainuiomata Camp pt2</a>
 				  </ul>
 				  </li>
 			     <li><a href="Contact.v1.html">Contact |</a>
 			    </ul> 
-            </div>		
-	
+            </div>	
 
 			<div>
 			<div class="box2"></div>
 			
 			<div class="box3"></div>
 			</div>
-			
-			<div class="sidebar">sidebar</div>

```

Comment:
I got rid of my sidebar as it was kind of useless because I didn't add anything on it.